auth.py

```diff
@@ -1,57 +1,35 @@
+import json
+import logging
+from datetime import datetime, timezone
 from functools import wraps
 from os import environ as env
-from typing import Optional
-from urllib.parse import quote_plus, urlencode
 
-from authlib.integrations.flask_client import OAuth
-from flask import Flask, redirect, session, url_for
-from werkzeug.local import LocalProxy
+import jinja2
+from flask import Flask, redirect, request, session, url_for
+from poprox_platform.aws import sqs
 from werkzeug.wrappers import Response
 
 from db.postgres_db import get_account, get_or_make_account
+from poprox_concepts.api.tracking import SignUpToken, to_hashed_base64
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+HMAC_KEY = env.get("POPROX_HMAC_KEY")
 
-oauth = OAuth()
 
 STATUS_REDIRECTS = {
     "new_account": "consent1",
-    "waiting_email_verification": "await_email_verification",
+    "pending_initial_preferences": "topics",
 }
 
 
 class Auth:
     def __init__(self, app: Flask) -> None:
-        oauth.init_app(app)
-        self.__registered_app = None
-
-    def __get_app(self) -> Optional[LocalProxy]:
-        if self.__registered_app is None:
-            self.__registered_app = oauth.register(
-                "auth0",
-                client_id=env.get("AUTH0_CLIENT_ID"),
-                client_secret=env.get("AUTH0_CLIENT_SECRET"),
-                client_kwargs={
-                    "scope": "openid profile email update:users",
-                },
-                server_metadata_url=f'https://{env.get("AUTH0_DOMAIN")}/.well-known/openid-configuration',
-            )
-        return self.__registered_app
-
-    def login(self):
-        return self.__get_app().authorize_redirect(
-            redirect_uri=url_for("callback", _external=True),
-        )
+        pass
```

Comment:
I usually write `pass` in empty methods too, but I recently learned that you can also put `...` and that works equivalently. (I'm not suggesting you do this, I just thought it was neat.)

Reply:
TIL